(Cyberiad/Коробка/Кибериада) Преработка всего сервисного отдела и дорм.

AyIong opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
**Ботаника**
-Добавлены створки.
-Добавлен холодильник возле стойки.
**Кухня**
-Добавлено 2 настенных аквариума.
**Бар**
-Прощайте стулья, привет удобные диваны.
-Дверь отныне двойная.
-Бармену добавлены файрлоки.
**Дормы**
Коридоры стали шире.
**Клоунская и мимская**
-Теперь они вертикальные.
**Туалет**
-Добавлена дополнительная кабинка.
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
https://discord.com/channels/617003227182792704/930740675350233098
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений
![HydroKitchen](https://user-images.githubusercontent.com/69762909/227376955-e1b9b954-eb88-46c5-9473-dbf248ca4cd5.png)
![BarToilet](https://user-images.githubusercontent.com/69762909/227376980-f64cc545-9c04-481e-b757-55bd27dc09f6.png)
![Dormitoriers](https://user-images.githubusercontent.com/69762909/227376989-51a2505b-210d-4ac5-9a7c-576853093bcb.png)

<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

AyIong committed:
(Cyberiad/Box/Кибериада) Ремап сервиса и дорм на коробке

AyIong committed:
(Cyberiad/Box/Кибериада) Ремап сервиса и дорм на коробке

AyIong committed:
Полировка ремапа.

AyIong committed:
Полировка ремапа.

dj-34 reviewed (changes requested):
Накакал трубой с initilize_directions замени на трубу без неё

![image](https://user-images.githubusercontent.com/20109643/227378575-5f4b7449-d346-4981-9bd1-59082d285373.png)

Накакал дубликатами с разницой в 1 пиксель, замени на те, которых больше на карте (pixel_y 24)

![image](https://user-images.githubusercontent.com/20109643/227378918-3f8f8b53-d710-4aa0-8c08-6993fe5c31dc.png)

Вот это похоже я забыл, тоже замени на south fire alarm

![image](https://user-images.githubusercontent.com/20109643/227379032-1ee85ed3-1351-4275-90dd-22166827cfab.png)

Очень-очень много неверных дубликатов, заменяй на те, которые должны быть

![image](https://user-images.githubusercontent.com/20109643/227379117-cf08c96f-79e8-480a-b73f-308d954ad4f6.png)

Опять накакал дубликатами

![image](https://user-images.githubusercontent.com/20109643/227379473-6c9c1fed-3eed-489c-aaf1-301ea2698a02.png)

Не верный старт цивилиана, видимо копировал из старой версии карт

![image](https://user-images.githubusercontent.com/20109643/227379717-1b0d8c41-7a50-46e8-80a6-0b8c749e1934.png)

Дубликаты

![image](https://user-images.githubusercontent.com/20109643/227379745-de535622-7999-400c-9cdf-4b8fd47853e7.png)

dj-34 commented:
UPD: dir 6 не использовать на камерах, есть dir 2, иначе будет жаловаться, что камера не находится на стене.

AyIong committed:
Фиксы by dj-34

AyIong committed:
Фиксы by dj-34

AyIong committed:
Небольшая перестановка мусоропровода

AyIong committed:
Небольшая перестановка мусоропровода

Dimach committed:
Revert "Русские и усиленные Стражи v.2.0." (#2411)

Revert "Tweak: adjust guardians stats (#2330)"

This reverts commit c83cfa2be25ce735d808da36d52d1f1339b07257.

AyIong committed:
Финальные правки

AyIong committed:
Финальные правки

Rerik007 committed:
fix: clockwork flash location (#2410)

lebedev committed:
fix: Forbid AI from yoinking magboots (#2406)

SAADf603 committed:
Fix: Doshi Co icon_state logic (#2395)

Dimach committed:
Fix: map rotate runtimes (#2425)

Rerik007 committed:
fix: fulton with nobeacon (#2412)

Daeberdir committed:
Fix: fix typo in #2327. (#2417)

Daeberdir committed:
Fix: Minor text improvements and corrections for new medical mecha modules. (#2418)

BeebBeebBoob committed:
Fix: Map: Cerestation (#2416)

Daeberdir committed:
Fix: Wrong way to fatcat holder sprite and cleaning 1 dot in glasses sprite. (#2404)

BeebBeebBoob committed:
Feat: Suicide Hud (#2407)

github-actions[bot] committed:
Automatic NanoMap Update (#2427)

NanoMap Auto-Update (Sat Mar 25 01:14:16 UTC 2023)

Co-authored-by: NanoMap Generation <action@github.com>
Co-authored-by: Dimach <Dimach@users.noreply.github.com>

lebedev committed:
feat: Anonimize butts photocopies (#2413)

Rinar21 committed:
FIX: Odysseus doesn't count syringe_gun_upgrade as a equipment (#2428)

ErdGinalD committed:
Feat: add craft for blood packs (#2401)

Rerik007 committed:
fix: moonlin drinks (#2414)

BeebBeebBoob committed:
Fix: Runtime Investigate log power alarm (#2125)

dependabot[bot] committed:
Build(deps): Bump express from 4.17.1 to 4.18.2 in /tgui (#1742)

Bumps [express](https://github.com/expressjs/express) from 4.17.1 to 4.18.2.
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/master/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.17.1...4.18.2)

---
updated-dependencies:
- dependency-name: express
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

lebedev committed:
feat: QoL improvement: copy AI tracking ability to nanomap (#2405)

AyIong committed:
Merge branch '5dayold' of https://github.com/Aylong220/Paradise into cyberiad-service-dorms-remap

Dimach commented:
Что за бардак в ченжах? 33 файла изменены, при этом они вообще никак к ПРу не относятся

AyIong commented:
> Что за бардак в ченжах? 33 файла изменены, при этом они вообще никак к ПРу не относятся

Облажался пока изучал гитхаб, в целом такого больше не должно повториться, спасибо что заметил. ПР закрываю, я понял где ошибся, в будущем постараюсь не допустить такого ужаса.
